modules/button/react/lib/DeleteButton.tsx

```diff
@@ -56,5 +56,7 @@ const DeleteButton: ButtonOrAnchorComponent<DeleteButtonProps> = ({
 );
 
 DeleteButton.Size = ButtonSize;
+// @ts-ignore ButtonOrAnchorComponent requires a Variant, but will complain Variable 'Variant' implicitly has an 'any' type. ts(7005)
+DeleteButton.Variant = undefined;
```

Comment:
I had to make `Variant` required in order for things like `Button.Variant.Small` to not throw "possibly undefined" errors. Rather than defining a brand new interface that didn't include `Variant`, I opted for the ability to set `V` to `undefined`.

Without this line, Typescript will thrown an error on the `const DeleteButton` line saying `Variant` is missing. So I have to set it. If I set it to anything other than `undefined`, Typescript will tell me `'foo' is not compatible with undefined`. If I set it to `undefined`, it will say `Variant` is implicitly `any`... This sets it to make Typescript happy, but disables the implicit any error...